src/Business Foundation/App/NoSeries/src/Setup/NoSeries.Page.al

```diff
@@ -180,12 +181,18 @@ page 456 "No. Series"
                     begin
                         Rec.TestField(Code);
                         NoSeriesSetupImpl.SetImplementation(Rec, Implementation);
+                        UpdateLineActionOnPage();
                     end;
                 }
             }
         }
         area(FactBoxes)
         {
+            part(NoSeriesRelationsFactBox; "No. Series Relationships Part")
```

Comment:
I was wondering if it also makes sense to add a part with the lines, thoughts?

Reply:
Could be a condensed version with just Starting No., Ending No., Last No. Used

Reply:
What is the use case? The lines are already displayed on the page.
![image](https://github.com/microsoft/BCApps/assets/34504100/f814364a-f69c-4cec-8d1b-6aef2caa60b4)

Reply:
might be nice to see the lines yes. Also, could add that same info but for 'current line' to the relationship part ... might have to do some background processing if its slow though...

Reply:
@NKarolak A line is displayed, based on Workdate. You likely have many lines. The part could show all open lines.

Reply:
Adding No. Series Lines part, adding all fields it currently has but hiding everything except the three fields Starting No., Ending No., Last No. Used. This allows customers to show any fields they need for their task.

Reply:
The page now looks like this:
<img width="865" alt="image" src="https://github.com/microsoft/BCApps/assets/10121161/1bdb0e6c-61b2-44b0-ae93-5e5b6e0af9f9">